app/shared/video-player/src/commonMain/kotlin/ui/progress/PlayerControllerBar.kt

```diff
@@ -536,6 +537,37 @@ object PlayerControllerDefaults {
             modifier = modifier,
         )
     }
+
+    @Composable
+    fun LeftBottomTips(
+        visible: Boolean,
+        onClick: () -> Unit,
+        modifier: Modifier = Modifier
+    ) {
+        AnimatedVisibility(
+            visible = visible,
+            enter = fadeIn(),
+            exit = fadeOut(),
+        ) {
+            Box(
+                modifier = modifier.clip(CircleShape)
+                    .background(MaterialTheme.colorScheme.surface.copy(alpha = .3f)),
+                contentAlignment = Alignment.Center,
```

Comment:
```suggestion
                contentAlignment = Alignment.CenterStart,
```
看起来这个地方应该用 CenterStart